blocks/datatable/datatable.js

```diff
@@ -0,0 +1,124 @@
+import { fetchPlaceholders } from '../../scripts/aem.js';
+
+// Fetch placeholders
+const placeholders = await fetchPlaceholders('');
+
+const {
+  sNo,
+  city,
+  country,
+  timezone,
+} = placeholders;
```

Comment:
Always go with standard way for block specific placeholders.

Ex: BLOCKNAME-KEY